spec.html

```diff
@@ -31031,9 +31040,9 @@ <h1>String.prototype.split ( _separator_, _limit_ )</h1>
           1. Let _p_ be 0.
           1. Let _q_ be _p_.
           1. Repeat, while _q_ &ne; _s_,
-            1. Let _e_ be SplitMatch(_S_, _q_, _R_).
-            1. If _e_ is *false*, set _q_ to _q_ + 1.
+            1. If ! StringIncludesAt(_S_, _R_, _q_) is *false*, then set _q_ to _q_ + 1.
             1. Else,
```

Comment:
since the "else" is multiline, i think the "if" needs to be too.

Reply:
The spec has about 37 examples of an 'inline' `If` followed by a 'multiline' `Else`. And about 18 examples of the reverse.

Reply:
Compared to how many examples are both multiline or both inline, those seem like cases we should make consistent.